fakefs.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.3"
   spec.add_development_dependency "rspec", "~> 3.1"
   spec.add_development_dependency 'rubocop', '~> 0.35.0'
+  spec.add_development_dependency 'bump'
```

Comment:
It would probably be a good idea to specify a version here. (`0.5.3` seems to be the most recent version on rubygems.)